app/scripts/views/confirm_reset_password.js

```diff
@@ -15,13 +14,13 @@ define([
   'views/mixins/resume-token-mixin',
   'views/mixins/service-mixin'
 ],
-function (Cocktail, _, ConfirmView, BaseView, Template, p, Session, AuthErrors,
+function (Cocktail, ConfirmView, BaseView, Template, p, Session, AuthErrors,
       ResendMixin, ResumeTokenMixin, ServiceMixin) {
   'use strict';
 
   var t = BaseView.t;
 
-  var SESSION_UPDATE_TIMEOUT_MS = 10000;
+  var LOGIN_MESSAGE_TIMEOUT_MS = 10000;
```

Comment:
> I just saw some failures when running 10x, not quite ready.

@shane-tomlinson I don't think we can get a consistent redirect to settings with 10 seconds, the same browser test dance takes ~20 seconds. I hope I'm wrong